includes/class-give-donor.php

```diff
@@ -1323,4 +1323,42 @@ public function is_address_exist( $current_address_type, $current_address ) {
 
 		return $status;
 	}
+
+	/**
+	 * Get First Name from the name of the donor.
+	 *
+	 * @since 2.0
+	 *
+	 * return string $first_name First Name of the donor
+	 */
+	public function get_first_name() {
```

Comment:
This PR should have waited for the merge of https://github.com/WordImpress/Give/pull/1885/ because @mehul0810 already has developed a function for this without the usage of `explode` https://github.com/WordImpress/Give/pull/1885/files#diff-089c5b17d291555b446391df90ce616fR1358

Reply:
@DevinWalker 

Okay